src/assets/i18n/en.json5

```diff
@@ -6761,4 +6761,6 @@
   "register-page.registration.aria.label": "Enter your e-mail address",
 
   "forgot-email.form.aria.label": "Enter your e-mail address",
+
+  "search-facet-option.update.announcement": "The page will be reloaded. selected filter: ",
```

Comment:
We should add a placeholder for the filter to this message.  This can be changed to be something like:
```
"search-facet-option.update.announcement": "The page will be reloaded. Filter {{ filter }} is selected.",
```

That allows us to avoid having to append the filter name later on..and it will clean up your other code.  See my next comment